tests/unit/feature_gates_test.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from feature_gates import FeatureGates
+
+
+@pytest.mark.serial
+def test_feature_gates():
+    with pytest.raises(ValueError, match="Unknown feature='foo'"):
+        FeatureGates.get("foo")
+
+    with pytest.raises(ValueError, match="Cannot set unknown feature='foo'"):
+        FeatureGates.set("foo", 42)
+
+    # presently, source mapping features are off by default:
```

Comment:
This feels like a test of the default and not a test of the feature gates. 
maybe just pull False and True out into a local variable `default_sourcemap_gate : bool = FeatureGates.get("sourcemap_enabled")` and then the asserts become `assert FeatureGates.get("sourcemap_enabled") is default_sourcemap_gate` or `not default_sourcemap_gate`

Reply:
bcf1d4704e2cbd8675bb72291e4f635472b32aec